cmd/clusterctl/pkg/client/cluster/upgrader_info_test.go

```diff
@@ -47,7 +47,7 @@ func Test_providerUpgrader_getUpgradeInfo(t *testing.T) {
 			name: "returns all the expected info",
 			fields: fields{
 				reader: test.NewFakeReader().
-					WithProvider("p1", clusterctlv1.CoreProviderType, "https://somewhere.com"),
+					WithProvider("p1", clusterctlv1.InfrastructureProviderType, "https://somewhere.com"),
```

Comment:
Just curious why these had to change to `InfrastructureProviderType`. Is `getUpgradeInfo` only meant for `InfrastructureProviderType`?

Reply:
I think I figured this out. I see there's validation logic that checks the name of the provider must be `cluster-api` if it is a `CoreProviderType`.